zpages/zpages.go

```diff
@@ -31,16 +31,31 @@
 package zpages // import "go.opencensus.io/zpages"
 
 import (
+	"fmt"
 	"net/http"
+	"sync"
+
+	"go.opencensus.io/internal"
 )
 
-// Handler is an http.Handler that serves the zpages.
+// Handler is deprecated: Use NewHandler.
 var Handler http.Handler
 
+var enableOnce sync.Once
+
 func init() {
+	Handler = NewHandler("")
+}
+
+// GetHandler returns a handler that serves the z-pages.
+func NewHandler(prefix string) http.Handler {
```

Comment:
We shouldn't have the prefix here.

It is coupling the responsibilities of a mux and a handler into a handler.

Reply:
I this this is preferable to using StripPrefix. It is much more readable, does not allocate a new request object, and does not try to fake the URL (like StripPrefix does).